src/core/client/stream/tabs/Comments/Stream/AllCommentsTab/AllCommentsTabContainer.tsx

```diff
@@ -264,6 +266,8 @@ export const AllCommentsTabContainer: FunctionComponent<Props> = ({
               disabled={isLoadingMore}
               aria-controls="comments-allComments-log"
               className={CLASSES.allCommentsTabPane.loadMoreButton}
+              data-keystop={true}
+              data-isLoadMore={true}
```

Comment:
Data props typically are written in kebab case:

```
data-key-stop
data-is-load-more
```

Reply:
I don't think that there is a convention.
`aria` props does not separate words:
```
aria-labelledby
aria-describedby
```
either way is fine for me though.